src/coreclr/System.Private.CoreLib/src/System/Threading/Thread.CoreCLR.cs

```diff
@@ -309,6 +309,15 @@ public void DisableComObjectEagerCleanup()
         [MethodImpl(MethodImplOptions.InternalCall)]
         public extern void Interrupt();
 
+        internal bool Abort(int timeout) =>
```

Comment:
These FCalls/QCalls can be on the `ControlledExecution` type, so that they are close to where they are used.